docs/guides/migration-from-2-to-3.md

```diff
@@ -135,7 +135,7 @@ Although the `<Box ... />` element has a `children` prop of `<div className="div
 rendered children of the element that the box component renders is a `<div className="box">...</div>`
 element.
 
-Prior enzyme v3, we would observe the following behavior:
+Prior enzyme v2, we would observe the following behavior:
```

Comment:
In fact this is correct as-is (altho it should be "Prior to enzyme v3") - prior to v2 is v1. Prior to v3 is v1 and v2, which is what this sentence is describing.